src/components/dialog/Dialog.scss

```diff
@@ -1,5 +1,8 @@
 .modal-container {
-  position: relative;
+  position: absolute;

```

Comment:
is it the only one possible way to center element?

Reply:
For sure not the only possible)